doc/tutorial/search/index.md

```diff
@@ -255,25 +307,54 @@ There are 25 hits.
 ```
 \endsolution
 
+## Controlling the search output
+
+When calling the search algorithm, a lazy range over seqan3::search_result objects is returned each representing a
+single hit. This means that merely calling the seqan3::search algorithm will do nothing except configuring the search
+algorithm based on the given search configuration, query and index. Only when iterating over the lazy search result
+range, the actual search for every query is triggered. We have done this automatically in the previous examples when
+printing the result to the seqan3::debug_stream which then invokes a range based iteration over the returned range or
+by using the std::ranges::distance algorithm. However, in many cases we want to access the specific positions and
+information stored in the seqan3::search_result object to proceed with our application. Since some information might be
+more compute intensive then others there is a way to control what the final search result object will contain.
+
+### Output configuration
+
+The behaviour of the search algorithm is further controlled through the
+\ref search_configuration_subsection_output "seqan3::search_cfg::output_*" configurations.
+The following output configurations exists:
+
+- seqan3::search_cfg::output_query_id
+- seqan3::search_cfg::output_reference_id
+- seqan3::search_cfg::output_reference_begin_position
+- seqan3::search_cfg::output_index_cursor
+
+Similar to the max error configurations you can arbitrarily combine the configurations to customise the final output.
+For example, if you are only interested in the position of the hit within the reference sequence you can use the
+seqan3::search_cfg::output_reference_begin_position configuration. If instead, you need access to the index where the
```

Comment:
```suggestion
seqan3::search_cfg::output_reference_begin_position configuration. If instead you need access to the index where the
```

Reply:
Ok, so here I am struggeling as well. I would think it could be rewritten as: `Instead, if ...` such that follows `If, instead, ...`, am I right?

Reply:
I would go for `Instead, if you need to... hit was found, you can use`